terraform/cp4ba/variables.tf

```diff
@@ -70,36 +73,46 @@ variable "on_vpc" {
   description = "Select 'true' to install on a VPC cluster and it's using VPC Gen2. Note: CP4BA does not currently support VPC cluster."
 }
 
+variable "entitlement" {
+  default     = ""
+  description = "OCP entitlement: leave blank if OCP, set it to `cloud-pak` if cloud pak entitlement"
+}
+
+variable "vpc_zone_names" {
```

Comment:
This isn't relevant here since we don't support VPC.  I would comment out all VPC specific variables in the `variables.tf` file

Reply:
Done